packages/lit-dev-content/site/tutorials/content/async-directive/06.md

```diff
@@ -0,0 +1,54 @@
+The thing we want to do in the timer callback is to update the value rendered by
+the directive -- but notice the timer callback fires _outside_ of the normal
+directive `update`/`render` lifecycle where we normally return a value for
+the directive to render.
+
+Here, we can use `AsyncDirective`'s `setValue` API to update the directive's
```

Comment:
I think the name `setValue` is a little confusing and needs some explanation. What is a directive's "value"? It doesn't have `getValue` or a `value` property. Its arguments could be considered its value. 

Maybe just an `aside` that says that `setValue` sets the value of the Lit part the directive is rendered into or something?

Reply:
I think this also needs clarification in the directive docs.

Reply:
Reworded to "to update the result of the directive that is rendered into the template", and added a little more detail in the `custom-directives.md` page.